instructor_notes/00_instructor-setup.md

```diff
@@ -0,0 +1,12 @@
+When preparing for teaching this workshop, instructors should take the following setup steps:
+
+* [ ] Setup a non-personal trainer account (e.g., called "Data Lab") on your computer, if this doesn't already exist
+* [ ] Ensure the following softwares are installed and working in your trainer account:
+  * [ ] 1Password
```

Comment:
If we are installing 1Password, I hesitate to call this a non-personal account. Having 1Password set up means that it will have access to ~ everything.

So I'm not actually sure we need a separate account at all? Or if we do, perhaps we should have a "dummy" github account that we use with it and _not_ set up 1Password.

```suggestion
* [ ] Setup a training account (e.g., called "Data Lab") on your computer, if this doesn't already exist
* [ ] Ensure the following softwares are installed and working in your training account:
  * [ ] 1Password
```

Reply:
Something that will help _a bit_ is that "non-personal" is absolutely typo and should have said _personal, non-work_!

The separate account helps me with desktop clutter and definitely.definitely getting notifications off, b/c I'm paranoid, but I don't think it's totally necessary! I can change this to just say make sure you have these things installed, and if you are into it, make a separate non-work account for training only.